spec/features/single_content_item_spec.rb

```diff
@@ -4,14 +4,36 @@
     {}
   end
 
-  it 'renders the metric for unique_pageviews' do
+  before do
     content_api_has_metric('base/path',
                            'unique_pageviews',
                            '2000-01-01',
                            '2050-01-01',
                            unique_pageviews: { total: 145_000 })
+    content_api_has_timeseries('base/path',
```

Comment:
Sorry I didn't pick this up before, but can you rename `content_api_has_timeseries` and `content_api_has_metric` to `content_data_api_has*`?